src/operator/tensor/elemwise_binary_broadcast_op_basic.cc

```diff
@@ -11,7 +11,23 @@ namespace mxnet {
 namespace op {
 MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_add)
 .add_alias("broadcast_plus")
-.describe(R"code(Add arguments, element-wise with broadcasting.
+.describe(R"code(Returns element-wise sum of the input arrays with broadcasting.
+
+Both `broadcast_add` and `broadcast_plus` behave and work the same.
```

Comment:
here "behave and work" the words "behave" and "work" appear to have the exact same meaning. Instead just say

"The functions `broadcast_add` and `broadcast_plus` are identical."

If this statement is true, then why do we have have two functions? If they are identical why don't we just alias one to the other? And which one is preferred. If one is preferred, we should deprecate the other.